test/examples/user_defined_rules.erl

```diff
@@ -1,9 +1,9 @@
 -module(user_defined_rules).
 
--export([rule/3]).
+-export([rule/2]).
 
 -hank([unnecessary_function_arguments]).
 
--spec rule(any(), any(), any()) -> [elvis_result:item(), ...].
-rule(_Config, _Target, _) ->
+-spec rule(any(), any()) -> [elvis_result:item(), ...].
+rule(_Config, _Target) ->
```

Comment:
This is something we'll have to make work in the current implementation iteration (2/3 args) but I wanna have a good strategy for this.